app/src/main/java/com/bc/gordiansigner/helper/Bip39Words.kt

```diff
@@ -0,0 +1,2054 @@
+package com.bc.gordiansigner.helper
+
+object Bip39Words {
+    val valid = listOf(
```

Comment:
Minor comment. I think using the name `valid` is not really good. I prefer putting it outside if the object, just in a package level. Something like this:

```
package com.bc.gordiansigner.helper

val bip39Words = listOf(//stuffs)
```

or you can consider putting the words in the `asset` folder then load it when needed.

Reply:
Just shadowing the ios version: https://github.com/Fonta1n3/GordianSigner-Catalyst/blob/master/GordianSigner/Helpers/Bip39Words.swift
I put inside an object so that in the future, other users can trace it easier by finding the object name.

Reply:
How about something like this?
```
object Bip39 {
val words = listOf()
}
```